src/compiler/checker.ts

```diff
@@ -3106,7 +3128,7 @@ namespace ts {
             setObjectTypeMembers(type, members, callSignatures, constructSignatures, stringIndexType, numberIndexType);
         }
 
-        function resolveObjectOrUnionTypeMembers(type: ObjectType): ResolvedType {
+        function resolveStructuredTypeMembers(type: ObjectType): ResolvedType {

```

Comment:
Is there already a `resolveTypeMembers`? If so, what's the difference? If not, can we name this function that?